composer.json

```diff
@@ -7,7 +7,7 @@
     "license": "Apache-2.0",
     "require": {
         "php": "^7.4|^8.0",
-        "google/auth": "^1.33",
+        "google/auth": "dev-add-universe-domain-cache as 1.39",
```

Comment:
Needs to be updated once https://github.com/googleapis/google-auth-library-php/pull/533 is merged